### PR TITLE
#163519120 Add like and dislike a comment

### PR DESCRIPTION
**Pull request by filschristian**
#### What does this PR do?
Add like and dislike functionality on a comment

#### Description of Task to be completed?
An authenticated user can like or dislike a comment
A user can be able to see the users who liked or disliked a comment

#### How should this be manually tested?

- Pull this branch

- run `npm install`

- run `npm test`

- run `npm start` 

- read an article and like or dislike a comment

#### Any background context you want to provide?
You must be authenticated to be able to like or dislike a comment

#### What are the relevant pivotal tracker stories?
#163519120

#### Screenshots (if appropriate)
image

#### Questions:
N/A

### CODE REVIEW

**Comment by filschristian** [comment]
Your suggestions on improvement are welcome.
I am currently working on tests

**Review by Nziranziza — APPROVED** [review]

File: src/_styles/pages/_Article.scss [hunk 1]
```diff
@@ -185,4 +185,138 @@
...
+  position: absolute;
+  max-height: 100px;
+  width: 200px;
+  background-color: white;
+  border: 1px;
+  margin-bottom: -115px;
+  overflow: scroll;
+  box-shadow: 2px 2px 2px 2px #ccc;
+}
+.comment-action-count:hover .comment-likers {
```
Comment (NkFab) [inline src/_styles/pages/_Article.scss]: Try to be consistent with the scss systax.

File: src/_styles/pages/_Article.scss [hunk 2]
```diff
@@ -185,4 +185,138 @@
...
+  }
+  .article-comment__center {
+    margin: auto;
+    width: 60%;
+    font-family: Roboto-light;
+    @media screen and (max-width: $tablet-width) {
+      width: 90%;
+    }
+  }
+}
```
Comment (NkFab) [inline src/_styles/pages/_Article.scss:274]: After each main class selectors put a new line to make the code more readable. This is one instance of this case look for others.

Comment (filschristian) [reply src/_styles/pages/_Article.scss:274]: Feedback Implemented. Thanks

**Review by NkFab — APPROVED** [review]

File: src/__tests__/components/Comment.test.js [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+    expect(mockFn).toHaveBeenCalled();
+  });
+
+  describe('onEnterPress', () => {
+    const event = {
+      keyCode: 13,
+      shiftKey: false,
+      preventDefault: jest.fn(),
+      target: {
+        value: ' ',
```
Comment (kimotho-njoki) [inline src/__tests__/components/Comment.test.js]: is `value` an empty string or `space`? There's a difference

Comment (filschristian) [reply src/__tests__/components/Comment.test.js]: This is a space because we wanted to test trimming.

**Pull request merged**